### PR TITLE
Add Native Image Demo how to Build a Native Executable with Reflection.

### PR DESCRIPTION
**Pull request by LesiaChaban**

### CODE REVIEW

File: native-image-reflection-example/ReflectionExample.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (olyagpl) [inline native-image-reflection-example/ReflectionExample.java:1]: Empty line at the beginning.

File: native-image-reflection-example/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+# Native Image Support for Java Reflection
```
Comment (olyagpl) [inline native-image-reflection-example/README.md:1]: @LesiaChaban, please add an entry in the top-level README for this demo.

File: native-image-reflection-example/ReflectionExample.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    static String capitalize(String input) {
+        return input.toUpperCase();
+    }
+}
+
+public class ReflectionExample {
+
+    public static void main(String[] args) throws ReflectiveOperationException {
+        if (args.length == 0) {
+            System.err.println("You must provide the name of a class, the name of its method and input for the method");
```
Comment (fniephaus) [inline native-image-reflection-example/ReflectionExample.java:21]:
```suggestion
            System.err.println("Please provide a class name, a method name, and input for the method (for example: StringCapitalizer capitalize \"hello\")");
```

**Review by fniephaus — APPROVED** [review]
LGTM, thanks! I added one suggestion.

**Pull request merged**